e2etest/l2tests/l2.go

```diff
@@ -392,8 +392,11 @@ var _ = ginkgo.Describe("L2", func() {
 			ginkgo.By("creating a service")
 			svc, _ := service.CreateWithBackend(cs, f.Namespace.Name, "external-local-lb", testservice.TrafficPolicyCluster)
 			defer func() {
-				err := cs.CoreV1().Services(svc.Namespace).Delete(context.TODO(), svc.Name, metav1.DeleteOptions{})
-				framework.ExpectNoError(err)
+				_, err := cs.CoreV1().Services(svc.Namespace).Get(context.TODO(), svc.Name, metav1.GetOptions{})
```

Comment:
Why you need this?

Reply:
Because i added in the end of the test the deletion of the service to validate the metrics. Once the defer is called it can't find the service (as it was already deleted) and the test is failing. I didn't want to remove the defer since the test can fail before getting to the deletion part, in such state we need the defer to delete the service.

Reply:
Then just ignore the error if it's not found, I think it will be clearer.